app/templates/components/valid-input.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="input-row">
+  <label class="label">{{label}} </label>
+  <div class="field">
+    {{#if textarea}}
+        {{textarea class=inputClassName value=value}}
+    {{else}}
+        {{input class=inputClassName value=value}}
+    {{/if}}
+  </div>
+  {{#if errorMessage}}
+    <div class="error-msg">
+      {{errorMessage}}
+    </div>
+  {{/if}}
+</div>
+
+{{yield}}
```

Comment:
@shenbaga-praburaj , this is really awesome 🥇 , just a minor change: 

Rename this component to `input-field` valid input don't match well. Its just a form component which has check for validation and show error message :)